Makefile

```diff
@@ -5,14 +5,6 @@ BINDIR ?= output
 
 include build/Configfile
 
-USE_VENDORIZED_BUILD_HARNESS ?=
-
-ifndef USE_VENDORIZED_BUILD_HARNESS
```

Comment:
The build is currently failing because this change block removes a make target that the travis build needs to succeed. I believe. It's good that we caught this missing..